src/HexagonOffload.cpp

```diff
@@ -259,7 +259,15 @@ class InjectHexagonRpc : public IRMutator {
 #else
         debug(1) << "Hexagon device code module: " << device_code << "\n";
         compile_module_to_llvm_bitcode(device_code, "hex.bc");
-        int result = system("${HEX_TOOLS}/bin/hexagon-clang hex.bc -fPIC -O3 -Wno-override-module -shared -o hex.so");
+
+        int result = 0;
+        string S64 = "${HEX_TOOLS}/bin/hexagon-clang hex.bc -fPIC -O3 -Wno-override-module -shared -o hex.so";
```

Comment:
I'd prefer to write this like:

    string hex_command = "${HEX_TOOLS}... "
    if (target.has_feature(Target::HVX_128)) {
        hex_command += " -mhvx-double";
    }
    int result = system(hex_command.c_str());